traffic_ops/traffic_ops_golang/traffic_ops_golang.go

```diff
@@ -46,6 +46,9 @@ import (
 // set the version at build time: `go build -X "main.version=..."`
 var version = "development"
 
+// Traffic Ops version, set before building
+const Version = "3.0.0"
```

Comment:
that "version" should not be named "version" imo - it describes the environment, not the actual version. AFAIK there's no previously-existing code that sets an actual version.

Reply:
`version` only reads `development` in development. The `build.sh` sets it to the real version: https://github.com/apache/trafficcontrol/blob/master/traffic_ops/build/traffic_ops.spec#L114

The overall Traffic Control version is maintained in https://github.com/apache/trafficcontrol/blob/master/VERSION - we don't want that in multiple places.

Reply:
If you want to see that in your dev builds, you can always build with `go build -ldflags "-X main.version=traffic_ops-$(cat ../../VERSION)"`

Reply:
hm, I suppose that was what I was looking for, then. Is there really no way to get the `go` toolchain to set that without explicitly passing it on the command line? e.g. in a Makefile you could just build that into the recipe, but a Makefile for a Go project doesn't make sense because `go` does everything `make` would do.

Reply:
>Is there really no way to get the go toolchain to set that without explicitly passing it on the command line?
Sort of. You could use a `go generate` tag, but that's still an additional build step. Not really worth the complexity for this one tiny use case.

> a Makefile for a Go project doesn't make sense because go does everything make would do.
But it doesn't. Again, there's `go generate`, but that's a separate command, which is an argument for using Make to `go generate; go build`.

Makefiles for Go projects do make sense. I don't think it's worth the cost here; but there are use cases for it.